variants/vueapp/.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Do we really want this? Are we not mainly using PyCharm?

Reply:
Was added by default, didn't notice

Reply:
removed